natlas-agent/natlas-agent.py

```diff
@@ -77,7 +77,7 @@ def main():
 
 	servicesSha = ""
 	BASEDIR = os.path.abspath(os.path.dirname(__file__))
-	SERVICESPATH = os.path.join(BASEDIR, "natlas-services")
+	SERVICESPATH = os.path.join(BASEDIR, 'tmp', "natlas-services")
```

Comment:
There's one problem with moving natlas-services to a `tmp/` folder. The `command_builder` references this file when it constructs the command telling nmap where the services file is.

https://github.com/natlas/natlas/blob/9eaa8ba2a793fce87913655f6f0afa858adeab2a/natlas-agent/natlas/threadscan.py#L20